android/app/src/main/java/com/dreamteam/tourneynizer/data/Tournament.java

```diff
@@ -0,0 +1,163 @@
+package com.dreamteam.tourneynizer.data;
+
+import android.graphics.Bitmap;
+import android.location.Address;
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+import android.widget.TextView;
+
+import java.io.Serializable;
+import java.sql.Time;
+
+/**
+ * Created by ryanl on 2/3/2018.
+ */
+
+public class Tournament implements Parcelable {
+
+    public static final Parcelable.Creator<Tournament> CREATOR
+            = new Parcelable.Creator<Tournament>() {
+        public Tournament createFromParcel(Parcel in) {
+            return new Tournament(in);
+        }
+
+        public Tournament[] newArray(int size) {
+            return new Tournament[size];
+        }
+    };
+
+    private long id;
+    private String name;
+    private String description;
+    private Address address;
+    private Time startTime;
+    private Time endTime;
+    private int maxTeams;
+    private int currentTeams;
+    private Time timeCreated;
+    private TournamentType tournamentType;
+    private Bitmap logo;
+    private int numCourts;
+    private long creatorUserID;
+    private boolean cancelled;
+
+    public Tournament(Tournament t) {
+        this(t.id, t.name, t.description, t.address, t.startTime, t.endTime, t.maxTeams, t.currentTeams, t.timeCreated, t.tournamentType, t.logo, t.numCourts, t.creatorUserID, t.cancelled);
+    }
+
+    public Tournament(long id, @NonNull String name, @NonNull Address address, @NonNull Time startTime, int maxTeams, int currentTeams, @NonNull Time timeCreated, @NonNull TournamentType tournamentType, int numCourts, long creatorUserID) {
+        this(id, name, null, address, startTime, null, maxTeams, currentTeams, timeCreated, tournamentType, null, numCourts, creatorUserID, false);
+    }
+
+    public Tournament(long id, @NonNull String name, String description, @NonNull Address address, @NonNull Time startTime, Time endTime, int maxTeams, int currentTeams, @NonNull Time timeCreated, @NonNull TournamentType tournamentType, Bitmap logo, int numCourts, long creatorUserID, boolean cancelled) {

```

Comment:
Maybe try to use the builder pattern instead of creating a bunch of long constructors?